src/main.rs

```diff
@@ -1376,13 +1377,126 @@ fn parse_manpage_at_path(
 #[derive(Copy, Clone, Debug)]
 struct Progress(pub bool);
 
-// TODO Arg/output types?
-fn parse_and_output_man_pages(
-    _paths: impl Iterator<Item = PathBuf>,
-    _output_directory: PathBuf,
-    Progress(_show_progress): Progress,
-) {
-    unimplemented!();
+// Output: Result<cmd_name, error>
+// TODO: Result<cmd_name, CompletionsError>
+fn parse_and_output_man_pages<P: AsRef<Path>>(
+    paths: impl Iterator<Item = P>,
```

Comment:
This could be simply `Vec<PathBuf>` or `Vec<Path>`.

Reply:
1f0458c